loops-and-arrays.js

```diff
@@ -5,7 +5,12 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
-
+let i = 0;
```

Comment:
Je mag i ook introduceren in de for loop, dit heet dan een ge-impliceerde variabele, het is wel netjes dat je i eerst als variabele declareert maar het hoeft niet. Alternatief:  je kan in de for loop ook een let stoppen: